go/common/sgx/ias/quote_test.go

```diff
@@ -38,12 +41,8 @@ func TestQuote(t *testing.T) {
 		"CPUSVN",
 	)
 	require.EqualValues(t, 0x00000000, quote.Report.MiscSelect, "MISCSELECT")
-	require.Equal(
-		t,
-		"07000000000000000700000000000000",
-		hex.EncodeToString(quote.Report.Attributes[:]),
-		"ATTRIBUTES",
-	)
+	require.EqualValues(t, 0x0000000000000007, quote.Report.Attributes.Flags, "ATTRIBUTES.FLAGS")
```

Comment:
God I hope that we don't have XFRM and the flags transposed.

Reply:
We don't ;)